### PR TITLE
Explain where rendering is

### PR DESCRIPTION
**Pull request by stepancheg**
It was not easy to find. Add some pointers to the comment.

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]
Helpful! It's still a bit weird to me that we're using this struct like a module-level doc comment, but this is a good addition.

**Comment by stepancheg** [comment]
My reasoning is this. I'm looking at various schedule labels like `PostUpdate` and `Last`. Do they perform rendering? Not clear, but they link to the `Main` schedule. So I go to `Main`, and this is the end, no more outgoing links.

Perhaps added wall of text can be moved to `PipelinedRenderingPlugin`, before this:

https://github.com/bevyengine/bevy/blob/d99053cc8ad08b36c9c2215c4e974bbb066ec35c/crates/bevy_render/src/pipelined_rendering.rs#L28-L38

File: crates/bevy_app/src/main_schedule.rs [hunk 1]
```diff
@@ -23,6 +23,17 @@ use bevy_ecs::{
...
+/// # Rendering
+///
+/// Note rendering is not executed in the main schedule by default.
+/// Instead, rendering is performed in a separate [`SubApp`](crate::app::SubApp)
+/// which exchanges data with the main app in between the main schedule runs.
+///
+/// When pipelined rendering is enabled, rendering is performed in a separate thread,
+/// concurrently with the main schedule, one rendering iteration for each main schedule iteration.
+///
+/// See `RenderPlugin` and `PipelinedRenderingPlugin` for more details.
```
Comment (mockersf) [inline crates/bevy_app/src/main_schedule.rs:36]: could you link those to docs.rs pages?

**Comment by stepancheg** [comment]
Added links to docs.rs, made test in `Main` shorter, added text to `RenderPlugin`.

**Review by NthTensor — APPROVED** [review]

**Pull request merged**